README.md

```diff
@@ -96,15 +96,17 @@ for the [Maps](https://github.com/JeroenDeDauw/Maps) and
 
 * The parsers no longer extend `StringValueParser`
 	* They no longer have public methods `setOptions` and `getOptions`
-	* They no longer have protected field `options`
-	* They no longer have protected methods `getOption`, `requireOption`, `defaultOption` and `stringParse`
+	* With a few exceptions, they no longer have protected field `options`
+	* With a few exceptions, they no longer have protected methods `getOption`, `requireOption`,
+	  `defaultOption` and `stringParse`
```

Comment:
What exceptions?

Reply:
LatLongParserBase and it's 4 subclasses still have a protected $options property, as well as a protected getOption() method.